pkg/flow/database.go

```diff
@@ -126,6 +126,16 @@ type nodeData struct {
 	path, dir, base string
 }
 
+func (d *nodeData) tags() map[string]string {
+	tags := make(map[string]string)
```

Comment:
If these keys are what the end user will have to use to filter their data we might want to create some more transparent, user-friendly names. This might not make sense for all keys, just ones the user could care most about. For example, `nd-path` could be a very common one, and it won't clash with anything if we just call it `path`. `ins-namespace` and `wf-namespace` conceptually can never be different. Instead we should just have `namespace`.